client/src/main/scala/com/aliyun/emr/rss/client/write/LifecycleManager.scala

```diff
@@ -104,6 +113,14 @@ class LifecycleManager(appId: String, val conf: RssConf) extends RpcEndpoint wit
   private val responseCheckerThread =
     ThreadUtils.newDaemonSingleThreadScheduledExecutor("rss-master-resp-checker")
 
+  private val changePartitionExecutors = ThreadUtils.newDaemonCachedThreadPool(
+    "lifecycle-manager-change-partition",
```

Comment:
since it lives in the Spark process, can we add an `rss-` prefix to the thread name?

Reply:
> since it lives in the Spark process, can we add an `rss-` prefix to the thread name?

Yea, done